src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessThread.cs

```diff
@@ -91,6 +91,7 @@ public ThreadPriorityLevel PriorityLevel
             [SupportedOSPlatform("windows")]
             [SupportedOSPlatform("linux")]
             [SupportedOSPlatform("freebsd")]
+            [SupportedOSPlatform("illumos")]
```

Comment:
`[SupportedOSPlatform("illumos")]`
Error `error CA1418: The platform 'illumos' is not a known platform name`

The way this analyzer is designed: https://github.com/dotnet/roslyn-analyzers/blob/62b0e094420a1f93d32ecb338aff8ad498ec9cd5/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/UseValidPlatformString.cs#L107, it requires us to have a public API for illumos https://github.com/dotnet/runtime/blob/01b4233dca6654663e2f7ea672f27386e537bc0c/src/libraries/System.Runtime/ref/System.Runtime.cs#L4898 which would require an API review process. It is an inconvenience for port work.

TL;DR We can't use illumos just yet, remove it for now.

```suggestion
```